source/FilterableLogMonitor.js

```diff
@@ -44,6 +49,12 @@ export default class FilterableLogMonitor extends Component {
     }
   }
 
+  componentDidUpdate () {
+    if (this.isAutoScrollEnabled) {
+      this.wrapperNode.scrollTop = this.containerNode.offsetHeight
+    }
+  }
```

Comment:
Should we also do this check for `componentDidMount` to handle the case where there are a lot of actions dispatches early in the app, before the filterable log view has mounted?